src/model/ControllerOutdoorAir.cpp

```diff
@@ -337,35 +285,46 @@ namespace model {
       }
     }
 
+    boost::optional<ThermalZone> ControllerOutdoorAir_Impl::humidistatControlZone() const {
+      return getObject<ModelObject>().getModelObjectTarget<ThermalZone>(OS_Controller_OutdoorAirFields::HumidistatControlZoneName);
+    }
+
+    bool ControllerOutdoorAir_Impl::setHumidistatControlZone(const ThermalZone& thermalZone) {
+      bool result = setPointer(OS_Controller_OutdoorAirFields::HumidistatControlZoneName, thermalZone.handle());
+      result = result && setString(OS_Controller_OutdoorAirFields::HighHumidityControl, "Yes");
```

Comment:
The following 3 methods set HighHumidityControl to "Yes":
* setHumidistatControlZone
* setHighHumidityOutdoorAirFlowRatio
* setControlHighIndoorHumidityBasedOnOutdoorHumidityRatio

Resetting HumidistatControlZone does _not_ set HighHumidityControl back to "No" because the other 2 fields may be using it.

Reply:
Reading the I/O ref guide, it seems that set/resetHumidistatControlZone should manage the "HighHumidityControl" field in full.


> Field: High Humidity Control

> This choice field establishes whether or not the outdoor air flow rate is modified in response to high indoor relative humidity. Valid choices are Yes and No. If Yes is selected, the outdoor air flow rate may be modified when the indoor relative humidity is above the humidistat setpoint. **If No is selected, this option is disabled and the following three fields are not used.**

> Field: Humidistat Control Zone Name

> This input defines the zone name where the humidistat is located. This is the same name used in the [ZoneControl:Humidistat](https://bigladdersoftware.com/epx/docs/24-2/input-output-reference/group-zone-controls-thermostats.html#zonecontrolhumidistat) object. **This field is required when the input field High Humidity Control is specified as Yes**.

> Field: High Humidity Outdoor Air Flow Ratio
> [...]  **If this field is blank, the default value is 1** [...] 

> Field: Control High Indoor Humidity Based on Outdoor Humidity Ratio
> [...] **The default value is Yes**.
-----

Ideally:

* The setHumidityControl setter should not exist, only the getter.
* High Humidity Outdoor Air Flow Ratio should have been a required-filed and hardset to 1 in the Ctor.
* Control High Indoor Humidity Based on Outdoor Humidity Ratio should have been a required-field and hardset to "Yes/true" in the Ctor.